WooCommerce/Classes/ViewRelated/Dashboard/TopPerformers/TopPerformersDashboardViewModel.swift

```diff
@@ -74,8 +73,6 @@ final class TopPerformersDashboardViewModel: ObservableObject {
     func didSelectTimeRange(_ newTimeRange: StatsTimeRangeV4) {
         timeRange = newTimeRange
         saveLastTimeRange(timeRange)
-        usageTracksEventEmitter.interacted()
-        analytics.track(event: .Dashboard.dashboardMainStatsDate(timeRange: timeRange))
```

Comment:
This is removed because it's already tracked from the `TopPerformancersDashboardView` when selecting any item on the StatsTimeRangePicker.